### PR TITLE
new composition styling

### PR DESCRIPTION
**Pull request by mcginty**
anim

### CODE REVIEW

File: src/org/thoughtcrime/securesms/TransportOptionsAdapter.java [hunk 1]
```diff
@@ -54,6 +55,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
     ImageView       imageView = (ImageView) convertView.findViewById(R.id.icon);
     TextView        textView  = (TextView) convertView.findViewById(R.id.text);
 
+    DrawableCompat.setTint(imageView.getBackground(), transport.getBackgroundColor());
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/TransportOptionsAdapter.java]: Does this actually do something on GB?

**Comment by oaim** [comment]
I tried merging this in branch 2.21 and built the apk. The arrow and the paperclip have a white background on my phone (Android 4.1.2) so I can't see them. Everything else looks fine

**Comment by mcginty** [comment]
@oaim that's fixed now

**Comment by oaim** [comment]
Thanks! I also noticed that on dark theme I couldn't see what I was typing (I think I was typing white on white). I'll rebuild now ;)

**Comment by moxie0** [comment]
in 2.21.0

**Pull request closed without merging**